docs/code_ref/sunpy_stability.yaml

```diff
@@ -1,23 +1,23 @@
 coordinates:
   status: stable
-  comments: Should be stable, with additions and small changes expected.
+  comments: Any future changes should largely maintain backwards compatibility.
 data:
   status: mature
   comments: Should see no major changes.
 database:
   status: dev
-  comments: Undergoing changes with each major release.
+  comments: While it has not changed recently, it could be suggest to major revisions in future.
 image:
   status: dev
-  comments: Undergoing changes with each major release.
+  comments: While it has not changed recently, it could be suggest to major revisions in future.
 instr:
-  status: dev
-  comments: Maybe be phased out and migrated to affiliated packages in the future.
+  status: deprecated
+  comments: Will be removed in sunpy 3.1. Please use `sunkit-instruments`.
```

Comment:
Does this link properly?

Reply:
I need to check, I hope so.

Reply:
It does not.

Reply:
We should be able to use inter-sphinx to do this? Is instruments in our intersphinx?

Reply:
In theory it is but it didn't work.